dashboard/legacy/test/middleware/files_api_test_helper.rb

```diff
@@ -36,22 +36,22 @@ def list_objects
   end
 
   def get_object(filename, body = '', headers = {})
-    get "/v3/#{@endpoint}/#{@channel_id}/#{filename}", body, headers
+    get "/v3/#{@endpoint}/#{@channel_id}/#{CGI.escape(filename)}", body, headers
```

Comment:
revert changes to this file? the tests seem not to depend on these changes, and adding a 2nd/3rd method for escaping adds to the cognitive load